src/main/java/carpet/script/value/FunctionValue.java

```diff
@@ -228,15 +227,6 @@ public LazyValue lazyEval(Context c, Integer type, Expression e, Tokenizer.Token
         {
             retVal = returnStatement.retval;
         }
-        catch (ThrowStatement throwStatement) // might not be really necessary
-        {
-            retVal = throwStatement.retval;
-            rethrow = true;
-        }
-        if (rethrow)
-        {
-            throw new ThrowStatement(retVal);
```

Comment:
this was I guess to add potentially stack information, but never happen.

Reply:
Hmm... Wouldn't it need to catch `ExpressionException` then (don't really know, just asking)? May try to see what can I do with this then.

Reply:
Yeah, that's fine, just asking.